apps/menu-bar/src/modules/Storage.ts

```diff
@@ -112,3 +112,22 @@ export function resetApolloStore() {
   apolloClient.resetStore();
   storage.delete('apollo-cache-persist');
 }
+
+export type Log = { command: string; info: string };
+export class Logs {
+  private logs = this.get();
+
+  push(log: Log) {
+    this.logs.push(log);
+    if (Platform.OS === 'web') {
+      localStorage.setItem('logs', JSON.stringify(this.logs));
+    }
+  }
+
+  get(): Log[] {
+    if (Platform.OS === 'web') {
+      return JSON.parse(localStorage.getItem('logs') ?? '[]');
+    }
+    return this.logs ?? [];
+  }
+}
```

Comment:
In order for the logs to work across multiple windows on react-native web we need to use some sort of storage solution like localStorage